mmdet/core/mask/mask_target.py

```diff
@@ -40,6 +60,34 @@ def mask_target_single(pos_proposals, pos_assigned_gt_inds, gt_masks, cfg):
 
     Returns:
         Tensor: Mask target of each positive proposals in the image.
+
+    Example:
+        >>> from mmdet.core.mask.mask_target import *  # NOQA
+        >>> import mmdet
+        >>> import mmcv
+        >>> from mmdet.core.mask import BitmapMasks
+        >>> H, W = 32, 32
+        >>> cfg = mmcv.Config({'mask_size': 8})
+        >>> rng = np.random.RandomState(0)
+        >>> # Masks for each ground truth box (relative to the image)
+        >>> #gt_masks_data = (rng.rand(3, H, W) > 0.5).astype(np.float)
```

Comment:
This line is redundant?

Reply:
Yes, I was debugging and I forgot to delete it.